cluster-logging/overlays/emea/jerry/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../base/
+  - ../../../../cluster-scope/base/operators.coreos.com/subscriptions/cluster-logging/
+  - ../../../../cluster-scope/base/operators.coreos.com/subscriptions/elasticsearch-operator/
```

Comment:
Please don't reference cross application. If manifests are part of the `cluster-scope` root folder, don't reference them from outside of the `cluster-scope`.